vizro-core/docs/pages/user-guides/custom-figures.md

```diff
@@ -1,70 +1,235 @@
 # How to create custom figures
 
-This guide explains how to create custom figures, which is useful when you need a figure that reacts to controls and actions.
+This guide explains how to create custom figures, which is useful when you need a component that reacts to
+[filter](filters.md) and [parameter](parameters.md) controls.
 
-There are two use cases where you might want to use custom figures:
+### When to use a custom figure
+You should use custom figures if **both** of the following conditions are met:
 
-* You need a figure that isn't available in our pre-defined figure functions [`vizro.figures`](../API-reference/figure-callables.md).
-* You need a figure that doesn't fit into the existing predefined components ([`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table] or [`AgGrid`][vizro.models.AgGrid]).
+- You need a figure that doesn't fit into the existing predefined components ([`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table] or [`AgGrid`][vizro.models.AgGrid]).
+- You need a figure that isn't available in our pre-defined figure functions [`vizro.figures`](../API-reference/figure-callables.md).
 
-Use the following approach, which is similar to creation of a [custom chart](../user-guides/custom-charts.md), do the following:
 
-1. Define a function that returns a `Dash` component.
-2. Decorate it with the `@capture("figure")` respectively.
+### Steps to create a custom figure
+
+1. Define a function that returns a [Dash component](https://dash.plotly.com/#open-source-component-libraries).
+2. Decorate it with `@capture("figure")`.
 3. The function must accept a `data_frame` argument (of type `pandas.DataFrame`).
-4. The figure should be derived from and require only one `pandas.DataFrame`. Additional dataframes from other arguments will not react to dashboard components such as `Filter`.
+4. The figure should be derived from and require only one `pandas.DataFrame`. Dataframes from other arguments
+will not react to dashboard controls such as [`Filter`](filters.md).
+
+The below examples can be used as a base to build more sophisticated figures.
+
+### Examples of custom figures
+
+#### Custom KPI card
+If you wish to change the design or content of our existing KPI (key performance indicator) cards from
+[`vizro.figures`](../API-reference/figure-callables.md), you can do so by following the steps described above.
+
+For instance, if you prefer the icon to be positioned on the right side of the title instead of the left,
+adjust the return statement of the function decorated with `@capture("figure")`.
```

Comment:
```suggestion
For instance, to make a KPI card with the icon positioned on the right side of the title instead of the left, copy and paste the [source code of `kpi_card`](API-reference/figure-callables/#vizro.figures.kpi_card) and
adjust the return statement of the function.
```